src/server/api/routes/questions.router.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const questionsController = require('../controllers/questions.controller');
+
+/**
+ * @swagger
+ * /questions:
+ *  get:
+ *    tags:
+ *    - Questions
+ *    summary: Get all questions
+ *    description:
+ *      Will return all questions.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  questionsController
+    .getQuestions()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /questions/{ID}:
+ *  get:
+ *    tags:
+ *    - Questions
+ *    summary: Get question by ID
+ *    description:
+ *      Will return single question with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the question to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/:id', (req, res, next) => {
+  questionsController
+    .getQuestionById(req.params.id)
+    .then((result) => res.json(result))
```

Comment:
I think user expecting single object but this code sending an array having single object as an element.

Reply:
I followed the modules example, as I believe this is what is expected as output. Anyone else want to offer an opinion? I'm happy to change it, but I'd rather wait until I hear some other opinions.

Reply:
I agree with Beth. I also think an array of questions make more sense to me :)

Reply:
Here I would expect a single object (question) returned not an array as it is one question we get